app/src/main/java/com/geeksville/mesh/service/MeshServiceNotifications.kt

```diff
@@ -165,10 +167,10 @@ class MeshServiceNotifications(
         )
     }
 
-    fun updateMessageNotification(name: String, message: String) =
+    fun updateMessageNotification(contactKey: String, name: String, message: String) =
         notificationManager.notify(
-            messageNotifyId,
-            createMessageNotification(name, message)
+            name.hashCode(), // show unique notifications,
```

Comment:
we could use `contactKey` instead of `name` here

Reply:
added that change in as well